README.md

```diff
@@ -1,2 +1,23 @@
 # webrtc-connectivity-check
```

Comment:
@tlvietdanh 

Break the structure as below

```
|- README.md --> describe in general what this repo is
|- apps
    |- client
         |- README.md
         |- index.js
    |- server
         |- src
              |- index.js
         |- docker
               |- Dockerfile
         |- README.md
         |- README.md
```

- In `client/README.md`
  * How end-user to setup --> need to be simplest
  * How to run --> need to be simplest
  * How to send back result --> need to be simplest
- In `server/README.md`
  * How to setup for code
  * How to deploy on server